photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -485,7 +485,8 @@ private Optional<EstimatedRobotPose> closestToCameraHeightStrategy(PhotonPipelin
                                             .transformBy(target.getBestCameraToTarget().inverse())
                                             .getZ());
 
-            if (alternateTransformDelta < smallestHeightDifference) {
+            if (alternateTransformDelta < smallestHeightDifference
+                    && alternateTransformDelta < bestTransformDelta) {
```

Comment:
So the point of this function is to find a transform such that the Z difference is minimized right? I think we can better represent this intent with something similar to (but not necessarily a) list comprehension? Probably out of scope, and I don't think anyone uses this function anyways.